eth/handler_test.go

```diff
@@ -565,7 +565,10 @@ func TestBroadcastBlock(t *testing.T) {
 		{100, 10},
 	}
 	for _, test := range tests {
-		testBroadcastBlock(t, test.totalPeers, test.broadcastExpected)
+		t.Run(fmt.Sprintf("%d-%d", test.totalPeers, test.broadcastExpected), func(t *testing.T) {
```

Comment:
I think this breaks something...
my diff: 
```diff
diff --git a/eth/handler_test.go b/eth/handler_test.go
index 84724f570..75aadea52 100644
--- a/eth/handler_test.go
+++ b/eth/handler_test.go
@@ -573,6 +573,8 @@ func TestBroadcastBlock(t *testing.T) {
 }
 
 func testBroadcastBlock(t *testing.T, totalPeers, broadcastExpected int) {
+
+       fmt.Printf("%v %v\n", totalPeers, broadcastExpected)
        var (
                evmux   = new(event.TypeMux)
                pow     = ethash.NewFaker()
```
produces on master
```
=== RUN   TestBroadcastBlock
1 1
2 1
3 1
4 2
5 2
9 3
12 3
16 4
26 5
100 10
--- PASS: TestBroadcastBlock (20.05s)
``` 
this branch:
```
=== RUN   TestBroadcastBlock
=== RUN   TestBroadcastBlock/1-1
=== PAUSE TestBroadcastBlock/1-1
=== RUN   TestBroadcastBlock/2-1
=== PAUSE TestBroadcastBlock/2-1
=== RUN   TestBroadcastBlock/3-1
=== PAUSE TestBroadcastBlock/3-1
=== RUN   TestBroadcastBlock/4-2
=== PAUSE TestBroadcastBlock/4-2
=== RUN   TestBroadcastBlock/5-2
=== PAUSE TestBroadcastBlock/5-2
=== RUN   TestBroadcastBlock/9-3
=== PAUSE TestBroadcastBlock/9-3
=== RUN   TestBroadcastBlock/12-3
=== PAUSE TestBroadcastBlock/12-3
=== RUN   TestBroadcastBlock/16-4
=== PAUSE TestBroadcastBlock/16-4
=== RUN   TestBroadcastBlock/26-5
=== PAUSE TestBroadcastBlock/26-5
=== RUN   TestBroadcastBlock/100-10
=== PAUSE TestBroadcastBlock/100-10
=== CONT  TestBroadcastBlock/1-1
100 10
=== CONT  TestBroadcastBlock/9-3
100 10
=== CONT  TestBroadcastBlock/4-2
100 10
=== CONT  TestBroadcastBlock/26-5
100 10
=== CONT  TestBroadcastBlock/100-10
100 10
=== CONT  TestBroadcastBlock/16-4
100 10
=== CONT  TestBroadcastBlock/12-3
100 10
=== CONT  TestBroadcastBlock/3-1
=== CONT  TestBroadcastBlock/2-1
100 10
100 10
=== CONT  TestBroadcastBlock/5-2
100 10
--- PASS: TestBroadcastBlock (0.00s)
    --- PASS: TestBroadcastBlock/12-3 (2.03s)
    --- PASS: TestBroadcastBlock/9-3 (2.03s)
    --- PASS: TestBroadcastBlock/3-1 (2.03s)
    --- PASS: TestBroadcastBlock/4-2 (2.03s)
    --- PASS: TestBroadcastBlock/2-1 (2.03s)
    --- PASS: TestBroadcastBlock/100-10 (2.03s)
    --- PASS: TestBroadcastBlock/26-5 (2.04s)
    --- PASS: TestBroadcastBlock/1-1 (2.04s)
    --- PASS: TestBroadcastBlock/5-2 (2.03s)
    --- PASS: TestBroadcastBlock/16-4 (2.04s)
PASS
ok      github.com/ethereum/go-ethereum/eth     (cached)
```
See also: https://gist.github.com/posener/92a55c4cd441fc5e5e85f27bca008721

Reply:
Oh my, classic

Reply:
Nice catch, thanks!